railties/lib/rails/api/task.rb

```diff
@@ -69,7 +73,12 @@ class Task < RDoc::Task
             README.rdoc
             lib/**/*.rb
           ),
-          exclude: "lib/rails/generators/rails/**/templates/**/*.rb"
+          exclude: %w(
+            lib/rails/generators/rails/**/templates/**/*.rb
+            lib/rails/test_unit/*
+            lib/rails/test_help.rb
```

Comment:
Not sure we should hide this. It's common for an app's `test_helper.rb` to require `test_help`